docker-compose/.env

```
@@ -15,6 +15,7 @@ WALLET_BACKEND_PORT=7001
 ISSUER_API_PORT=7002
 VERIFIER_API_PORT=7003
 VERIFIER_API2_PORT=7004
+ISSUER_API2_PORT=7005
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ISSUER_API2_PORT key should go before the ISSUER_API_PORT key

(UnorderedKey)